### ISSUE TITLE
Security_Log_Profile -> botDefence new REQUIRED  but not supported in AS3 values introduced in BIG-IP 14+ 

### ISSUE THREAD

**Issue by zhurvla** [F5Networks/f5-appsvcs-extension#270]
### Environment
 * Application Services Version: AS3.18 and AS3.20 pre-release
 * BIG-IP Version: 14.1.0.3 and newer

### Summary
One of the following flags must be enabled in  Security_Log_Profile -> botDefence
TMSH (valuse including old and supported and new unsupported):
root@(ltm-232)(cfg-sync Standalone)(LICENSE EXPIRES IN 4 DAYS:ModuleNotLicensed::Acti*)(/Common)(tmos)# modify  security log profile /Common/Shared/Log_Bot_Defense                                                                     _ bot-defense add { f5demo_log_all { filter {
Properties:
  log-alarm                            log-captcha-challenged-requests      log-legal-requests                   log-suspicious-browser
  log-block                            log-challenge-failure-request        log-malicious-bot                    log-tcp-reset
  log-bot-signature-matched-requests   log-challenged-requests              log-mobile-application               log-trusted-bot
  log-browser                          log-device-id-collection-request     log-none                             log-unknown
  log-browser-verification-action      log-honey-pot-page                   log-rate-limit                       log-untrusted-bot
  log-captcha                          log-illegal-requests                 log-redirect-to-pool                 }

GUI not able to create a profile without selecting one of the values, all not supported in AS3 currently:
Human Users: | Browser Mobile App
Bots: | Trusted Bot Untrusted Bot Suspicious Browser Malicious Bot
Unknown: | Enabled

### Steps To Reproduce
1. Submit the following declaration:
```json
{
    "class": "ADC",
    "schemaVersion": "3.17.0",
    "id": "urn:uuid:fcc90e7d-e24f-4419-ab6c-c0a4a3a8dc05",
    "label": "Converted Declaration",
    "remark": "Auto-generated by Project Charon",
    "Common": {
        "class": "Tenant",
        "Shared": {
            "class": "Application",
            "template": "shared",
            "Log_Bot_Defense_": {
                "application": {
                    "storageFilter": {
                        "logicalOperation": "and",
                        "requestContains": {
                            "searchIn": "search-in-headers",
                            "value": "Random value"
                        },
                        "requestType": "all"
                    }
                },
                "botDefense": {
                    "logBotSignatureMatchedRequests": true,
                    "logCaptchaChallengedRequests": true,
                    "logChallengedRequests": true,
                    "logLegalRequests": true,
                    "remotePublisher": {
                        "bigip": "/Common/local-db-publisher"
                    }
                },
                "dosApplication": {
                    "localPublisher": {
                        "bigip": "/Common/local-db-publisher"
                    }
                },
                "class": "Security_Log_Profile"
            }
        }
    }
}
```

### Expected Behavior
Config loaded

### Actual Behavior

"response": "01071b8b:3: Must choose at least one request type to log",

**Comment by dstokesf5** [comment]
Thank you for reporting this. It is being tracked in our internal product backlog as AUTOTOOL-577.

**Comment by dstokesf5** [comment]
This was resolved in the 3.26.0 release.
